registries/_format/time-local.md

```diff
@@ -0,0 +1,15 @@
+---
+owner: ya7010
+issue: 4759
+description: RFC3339 time without the timezone component
+base_type: string
+layout: default
+source_label: TOML
+source: https://toml.io/en/v1.0.0#local-time
```

Comment:
This is defined in [RFC3339, section 5.6](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6) as `partial-time` and not specific to TOML. Please adjust source reference and remove `source_label`.

Reply:
fix: 1f4191427576b299dbd61068c7733bbca02f67af

Reply:
fix: source link https://www.rfc-editor.org/rfc/rfc3339#section-5.6